app/pdfs/export_pdf.rb

```diff
@@ -0,0 +1,28 @@
+class ExportPdf < Prawn::Document
+  def initialize(curiosity)
+    super()
+    @curiosity = curiosity
+    head_curiosity
+    images
+    content
+  end
+
+  def head_curiosity
```

Comment:
Does this methods have to be public? If not, let's make them private

Reply:
that's right, they should be private, thanks :D